### PR TITLE
Style edits

### LINKED ISSUE #2: Design edits
#689F38 - dark primary green, #8BC34A - primary green, #607D8B - accent color, #212121 - primary text, #DCEDC8 - light primary color, #757575 - secondary text

### LINKED ISSUE #3: Tap event dependency breaks build
The dependency `react-tap-event-plugin` breaks the `npm start` build process.

Commenting out lines 8 and 9 of App.js allows build process to run.

```
// import injectTapEventPlugin from 'react-tap-event-plugin';
// injectTapEventPlugin();
```

### PR DESCRIPTION
**Pull request by rickymetz**
Closes #2 
Closes #3

### CODE REVIEW

**Comment by rickymetz** [comment]
Pre-existing issues:
-  Lines [8-9](https://github.com/NYCEDU/nycedu-react/pull/4/files#diff-14b1e33d5bf5649597cdc0e4f684daddR8) are commented out because the react-tap-event dependency break the build.
-  Slideout menu is broken. Unsure of why.

File: src/App.js [hunk 1]
```diff
@@ -9,42 +9,83 @@ import injectTapEventPlugin from 'react-tap-event-plugin';
...
+import iconTwitter from './icon-twitter.svg';
 
 class App extends Component {
 
   state = {
     open: false
   }
 
-  handleToggle(){
+  handleToggle = () => {
```
Comment (seanperkins) [inline src/App.js:24]: Thanks for updating these to use es6 binding

File: package.json [hunk 2]
```diff
@@ -9,13 +9,14 @@
...
+    "lodash": "^4.17.2",
+    "material-ui": "^0.16.3",
     "moment": "^2.14.1",
-    "react": "^15.3.1",
+    "react": "^15.4.0",
     "react-document-title": "^2.0.2",
     "react-dom": "^15.3.1",
     "react-router": "^2.7.0",
-    "react-tap-event-plugin": "^1.0.0"
+    "react-tap-event-plugin": "^2.0.0"
```
Comment (seanperkins) [inline package.json:19]: You updated this due to the breaking react changes?

File: src/home/Home.css [hunk 3]
```diff
@@ -32,12 +32,20 @@
 
 .project {
   text-align: center;
+
```
Comment (seanperkins) [inline src/home/Home.css]: whitespace

**Review by seanperkins — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (rickymetz) [reply package.json:19]: Yes, that's what was broken.

File: src/App.js (hunk 1 above)
Comment (rickymetz) [reply src/App.js:24]: 👍

**Pull request merged**